cabi/include/symbolic.h

```diff
@@ -28,7 +28,7 @@ enum SymbolicErrorCode {
   SYMBOLIC_ERROR_CODE_OBJECT_ERROR_UNSUPPORTED_OBJECT = 2101,
   SYMBOLIC_ERROR_CODE_OBJECT_ERROR_BAD_BREAKPAD_OBJECT = 2102,
   SYMBOLIC_ERROR_CODE_OBJECT_ERROR_BAD_ELF_OBJECT = 2103,
-  SYMBOLIC_ERROR_CODE_OBJECT_ERROR_BAD_MACH_OOBJECT = 2104,
+  SYMBOLIC_ERROR_CODE_OBJECT_ERROR_BAD_MACH_O_OBJECT = 2104,
```

Comment:
not sure if this will cause any problems?

Reply:
I was wondering the same...

Reply:
I manually reverted this. Although its an auto-generated file, so people will run into this again at some point.

Reply:
This is indeed strange. It would be a breaking change in the C-ABI, and it will generate a different name for the matching error in the Python layer. I think you can fix this with some sort of configuration for `cbindgen`, although the new version is actually correct.